Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -1,5 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
+from RecoLocalCalo.HcalRecAlgos.hcalRecAlgoESProd_cfi import *
```

Comment:
I've started jenkins tests, but I will likely ask for this to be removed after checking the state of this locally